src/DAL/index.ts

```diff
@@ -15,15 +18,28 @@ const pool = new Pool({
 });
 
 pool.on('connect', () => {
-    console.log('initiated db connection pool');
+    logger.info({
```

Comment:
where is the workflow?

Reply:
in the created log there is the workflow

Reply:
because there is only one flow in this service